src/cdk/config.bzl

```diff
@@ -13,6 +13,7 @@ CDK_ENTRYPOINTS = [
     "listbox",
     "menu",
     "observers",
+    "observers/private",
```

Comment:
Why do we need to make this private? I can see it being useful for users.

Reply:
I'd like to open it up eventually, but Jeremy wants to keep it private for the moment